app/src/main/java/com/hedvig/app/feature/insurance/ui/terminatedcontracts/TerminatedContractsActivity.kt

```diff
@@ -48,7 +48,7 @@ class TerminatedContractsActivity : BaseActivity(R.layout.terminated_contracts_a
             toolbar.applyStatusBarInsets()
             recycler.applyNavigationBarInsets()
             toolbar.setNavigationOnClickListener { onBackPressed() }
-            val adapter = InsuranceAdapter(marketManager, model::load)
+            val adapter = InsuranceAdapter(marketManager, model::load, {})
```

Comment:
Is this {} in this case because the Terminated activity doesn't make use of that specific item in the RV? I guess it's a side effect of reusing this Adapter. Looks a bit weird but I guess we gotta just accept it.

Reply:
That's correct - cross-sells cannot happen in the `TerminatedContractsActivity`, and hence it felt pointless to give it a meaningful function to call here. It's entirely a side-effect of us sharing the adapter.